javatests/arcs/core/allocator/DefaultStorageKeyCreatorTest.kt

```diff
@@ -0,0 +1,83 @@
+package arcs.core.allocator
+
+import arcs.core.common.Id
+import arcs.core.data.CreatableStorageKey
+import arcs.core.data.Plan
+import arcs.core.host.MultiplePersonPlan
+import arcs.core.host.PersonPlan
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import arcs.core.storage.driver.RamDisk
+import com.google.common.truth.Truth
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+@OptIn(ExperimentalCoroutinesApi::class)
+class DefaultStorageKeyCreatorTest {
+
+  @Before
+  open fun setUp() = runBlocking {
+    RamDisk.clear()
+    DriverAndKeyConfigurator.configure(null)
+  }
+
+  // Simple Case.
+  @Test
+  fun keyCreator_createStorageKeysIfNecessary() = creatorKeyTest(PersonPlan)
+
+  // Complex case.
+  @Test
+  fun keyCreator_createStorageKeysIfNecessary_manyParticles() = creatorKeyTest(MultiplePersonPlan)
+
+  /**
+   * Given a [Plan], test that it's keys start as [CreatableStorageKey]s and are transformed
+   * using the [DefaultStorageKeyCreator.createStorageKeysIfNecessary] method. Verify that the plan
+   * returned
```

Comment:
Verify that the plan returned what?

Reply:
Oops, apparently my train of thought just trailed off. Updated the comment.